charts/node/values.yaml

```diff
@@ -102,8 +102,11 @@ node:
   serviceMonitor:
     enabled: false
     #namespace: monitoring
-    #interval: 10s
-    # scrapeTimeout: 10s
+    interval: 1m
+    scrapeTimeout: 30s
+    targetLabels:
```

Comment:
setting this up will be a bit tricky and would probably be good to mention here: one can only target labels that have been added either by the `serviceLabels` or `extraLabels`.

Reply:
I added the details in the description for the parameters in  README.md